### ISSUE TITLE
cmake requires pkg_config for Python bindings on WIN32

### ISSUE THREAD

**Issue by masskilograms** [KDAB/KDDockWidgets#68]
Following errors occurs while running cmake on Windows with  `-DOPTION_BUILD_PYTHON_BINDINGS=ON`
The error is due to missing `pkg_config`

```
CMake Error at C:/Program Files/CMake/share/cmake-3.17/Modules/FindPackageHandleStandardArgs.cmake:164 (message):
  Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.17/Modules/FindPackageHandleStandardArgs.cmake:445 (_FPHSA_FAILURE_MESSAGE)
  C:/Program Files/CMake/share/cmake-3.17/Modules/FindPkgConfig.cmake:45 (find_package_handle_standard_args)
  cmake/Python/FindShiboken2.cmake:20 (find_package)
  python/CMakeLists.txt:14 (find_package)
```

The error is due to `find_package(PkgConfig REQUIRED)` line in the following cmake files:
- KDDockWidgets/cmake/Python/FindPySide2.cmake 
- KDDockWidgets/cmake/Python/FindShiboken2.cmake 

Both cmake files run happily when 
```cmake
find_package(PkgConfig REQUIRED)
pkg_check_modules(SHIBOKEN2_PRIV shiboken2 QUIET)
```

is replaced with:

```cmake
if(NOT WIN32)
    find_package(PkgConfig REQUIRED)
    pkg_check_modules(SHIBOKEN2_PRIV shiboken2 QUIET)
endif()
```

So WIN32 specific logic is implemented in these files, but crash occurs due to pkg_config not found.

Suggested patch attached. Happy to submit a PR, if you think that's valid.
[win32_cmake_files.patch.txt](https://github.com/KDAB/KDDockWidgets/files/5165113/win32_cmake_files.patch.txt)

Tested on:
Windows 10 version 1909 / CMake 3.17.0 / cl.exe 19.25.28611 (x64) / Qt 5.15.0

Thanks for creating Python bindings for KDDockWidgets!

**Comment by winterz** [comment]
I'll take care of this.  We didn't catch it on our CI because we just happen to have pkgconfig installed on our Windows workers.

**Comment by winterz** [comment]
should be fixed in sha 63f8468f485fc52dfbf0d9679a3366a7dc5ce519
